src/Pages/Students/Students.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react';
+import { StudentsType } from '../../utils/types';
+import styles from './Students.module.css';
+import TitleElement from '../../Components/TitleElement/TitleElement';
+import EntryElement from '../../Components/EntryElement/EntryElement';
+import PullToRefresh from 'pulltorefreshjs';
+import setFetch from '../../utils/helper';
+
+export default function Students(): JSX.Element {
+  const [allStudents, setAllStudents] = useState<StudentsType[]>([]);
+  PullToRefresh.init({
+    mainElement: 'div',
+    onRefresh() {
+      setFetch('students', setAllStudents);
+    },
```

Comment:
nice!

Reply:
I thought the same. But then I thought maybe I'll use the helper.tsx for other helper functions later.
Thanks for the feedback☺️